README.md

```diff
@@ -0,0 +1,59 @@
+# KNWRobot - Robotics Control Library for SMU KNW2300
+Controling Arduino: MEGA 2560 
+	
+Author: Alexandria Hancock
+
+## Setup
+Before using the library, you must first setup your laptop to be able to compile code written using Arduino C++. You will then need to add additional libraries that the code uses behind the scenes (they are included in this repo). Follow these steps:
+
+1) Download the [Arduino IDE](https://www.arduino.cc/en/Main/Software) for your operating system. Go through the installation steps.
+	* Windows: proceed with the default installation options. 
+	* Mac OSX: drag and drop the application into your Applications directory (most often `/Applications`).
+
+After finishing the installation, open the Arduino IDE. Continue selecting default options until you see a text editor with some empty functions. Close the application for now.
+
+2) [Download the contents of this repository](https://github.com/SMUKNW2300/arduino-library/archive/master.zip). Unzip the files onto your computer.
```

Comment:
I might change this to suggest downloading the KNW2300 Library zip file from the Releases section of the repo instead, as the Master branch had broken code in it several weeks ago when I created the library installation video guide.